app/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder.rb

```diff
@@ -43,47 +41,37 @@ def tag
     "<rubocop />"
   end
 
-  def build_messages
-    write_header
-    files.empty? ? write_success : write_offenses
-    @messages.collect!(&:string)
-  end
-
-  def write(line)
-    if message.length + line.length + 1 >= GITHUB_COMMENT_BODY_MAX_SIZE
-      @message = StringIO.new
-      @messages << message
-      write_header_continued
-    end
-
-    message.puts(line)
-  end
-
-  def write_header
+  def header
     commit_range = [
       branch.commit_uri_to(commits.first),
       branch.commit_uri_to(commits.last),
     ].uniq.join(" .. ")
-    write("#{tag}Checked #{"commit".pluralize(commits.length)} #{commit_range} with rubocop #{rubocop_version}")
+    header1 = "Checked #{"commit".pluralize(commits.length)} #{commit_range} with rubocop #{rubocop_version}"
 
     file_count    = results.fetch_path("summary", "target_file_count").to_i
     offense_count = results.fetch_path("summary", "offense_count").to_i
-    write("#{file_count} #{"file".pluralize(file_count)} checked, #{offense_count} #{"offense".pluralize(offense_count)} detected")
+    header2 = "#{file_count} #{"file".pluralize(file_count)} checked, #{offense_count} #{"offense".pluralize(offense_count)} detected"
+
+    "#{tag}#{header1}\n#{header2}"
   end
 
-  def write_header_continued
-    write("#{tag}**...continued**\n")
+  def continuation_header
+    "#{tag}**...continued**\n"
+  end
+
+  def build_messages
+    @message_builder = MiqToolsServices::Github::MessageBuilder.new(header, continuation_header)
+    files.empty? ? write_success : write_offenses
   end
 
   def write_success
-    write("Everything looks good. #{SUCCESS_EMOJI.sample}")
+    message_builder.write("Everything looks good. #{SUCCESS_EMOJI.sample}")
   end
 
   def write_offenses
     files.each do |f|
-      next if f["offenses"].empty?
```

Comment:
This was removed because it's already handled by https://github.com/ManageIQ/miq_bot/pull/60/files#diff-ec9d6040f291f72a47251ead8f0c2a00R79